src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Acquisition/MissingSetupComponentRegistrationService.cs

```diff
@@ -26,7 +26,8 @@ internal class MissingSetupComponentRegistrationService : IMissingSetupComponent
         .Add("v3.0", "Microsoft.NetCore.Component.Runtime.3.1")
         .Add("v3.1", "Microsoft.NetCore.Component.Runtime.3.1")
         .Add("v5.0", "Microsoft.NetCore.Component.Runtime.5.0")
-        .Add("v6.0", "Microsoft.NetCore.Component.Runtime.6.0");
+        .Add("v6.0", "Microsoft.NetCore.Component.Runtime.6.0")
+        .Add("v7.0", "Microsoft.NetCore.Component.Runtime.7.0");
```

Comment:
The title of this PR "Improve readability" suggests that no product change was made and that the PR just moves code around or adds documentation.

Here we are making a product change, and adding support for a new version of this component. For future, either pull this change out into a separate PR, or make sure it's clear in the PR description that such a change exists.